deploy/values.yaml

```diff
@@ -202,11 +215,24 @@ cmdesc:
   # -- Possible values are either 1 to enable all possible debugging, or a comma-separated list of debug tags (the tags printed before the log lines). No logging is specified as an empty string.
   verbosity: 1
 
-warehouse:
-  # -- Whether or not to enable the Warehouse component
-  ingester:
+historians:
+  uns:
     enabled: true
```

Comment:
I would consider carefully whether this should default to enabled. I think this will effectively double the historian storage requirement.

Reply:
I did notice this when testing. I think it should be disabled by default.